src/components/board/GargoyleEnemy.tsx

```diff
@@ -0,0 +1,40 @@
+import { useGLTF } from '@react-three/drei';
+import { Coordinates } from '../grid/SquareTypes';
+// @ts-ignore
+import gargoyleRedPath from '../objects/gargoyle_red.glb';

```

Comment:
I don't think there should be 2 components that do the same, but import a different color gargoyle. You can probably pass the `gargoyleRedPath` or `gargoyleBluePath` through components parameters. If you don't know the type of the imported object, just use `any`. It may not work, but I think it should